patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -121,6 +127,17 @@
     public boolean func_219970_i(ItemStack p_219970_1_) {
        return p_219970_1_.func_77973_b() == Items.field_222114_py;
     }
+@@ -306,4 +349,10 @@
++   @Deprecated
```

Comment:
When we add deprecation in forge to vanilla methods we put a comment that says what to use instead.